Network Design Guide/readme.md

```diff
@@ -1,14 +1,14 @@
 # Azure VMware Solution Landing Zone Accelerator Network Design Guide
 This guide covers network design for Azure VMWare Solution. It encompasses four design areas, summarized below.
 
-1. **Connectivity with on-premises datacenters**. Connectivity between Azure VMware Solution private clouds and on-prem sites supports a broad set of use cases, such as HCX migrations, hybrid applications, remote vSphere or NSX-T Data Center administration. Azure VMware Solution supports multiple options for hybrid connectivity, including Azure ExpressRoute circuits and internet-based IPSec VPNs. 
+1. **Connectivity with on-premises datacenters**. Connectivity between Azure VMware Solution private clouds and on-premises sites supports a broad set of use cases, such as HCX migrations, hybrid applications, remote vCenter or NSX-T Manager administration. Azure VMware Solution supports multiple options for hybrid connectivity, including Azure ExpressRoute circuits and internet-based IPSec VPNs. 
```

Comment:
NSX-T Data Center is the name of the software product that you can administer using NSX-T Manager. I would use "NSX-T Manager" in a sentence like "connect/log into NSX-T Manager" but not "administer NSX-T Manager" (you do not administer an administration console). Therefore, I think that the original version is OK.

Reply:
got it. Thanks for the explanation 👍